client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -426,6 +426,76 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: addNewSiteIcon,
 		},
+		{
+			name: 'manageSettingsWriting',
+			label: __( 'Manage writing settings' ),
+			searchLabel: _x( 'Manage blog writing settings', 'search label for command palette' ),
```

Comment:
I don't understand:

- What `searchLabel` does and why it's duplicate of `label`
- Why `searchLabel` is lowercase (I see https://github.com/Automattic/wp-calypso/pull/84489 but in the PR you don't explain _why_, only _what_)
- I added `_x()` to explain context better for translators

Reply:
> - What searchLabel does and why it's duplicate of label

The idea is the label is visible to the user and the searchLabel is just used to match the command search.
We can remove all the searchLabels if they match the visible Label. Thanks for pointing out.
I created an https://github.com/Automattic/dotcom-forge/issues/4629.
In your case I see you have the extra word `blog` there.

> - Why searchLabel is lowercase (I see https://github.com/Automattic/wp-calypso/pull/84489 but in the PR you don't explain why, only what)

Because the searchLabel is not visible to the user, it can be either lower or uppercase. It will be transformed to lowercase to match the search.

> - I added _x() to explain context better for translators

Thank you! We will do the same when needed.

Reply:
👍 Removed search labels.

I also added `context: [ '/settings' ]` — although I know the command palette isn't available outside `/sites` yet.

@sejas should I use `/settings/*` for context, or `/settings` suffices to cover all sub-pages?

Reply:
`/settings` should be enough. Currently we are making a string match. I'll probably need to make a change to match it.

We are planning to display all the commands in very page by default and just surface those that match the context based on the path.